tests/backends/oracle/test_operations.py

```diff
@@ -141,3 +141,16 @@ def test_sql_flush_sequences_allow_cascade(self):
         self.assertIn("BACKENDS_PERSON_SQ", statements[5])
         self.assertIn("BACKENDS_VERYLONGMODELN7BE2_SQ", statements[6])
         self.assertIn("BACKENDS_TAG_SQ", statements[7])
+
+    def test_last_executed_query_with_duplicate_params(self):
+        with connection.cursor() as cursor:
+            query = "UPDATE backends_square SET root = %s + %s WHERE id = %s"
+            params = [42, 42, 1]
+            cursor.execute(query, params)
+            last_executed_query = connection.ops.last_executed_query(
+                cursor, query, params
+            )
+            self.assertEqual(
+                last_executed_query,
+                "UPDATE backends_square SET root = 42 + 42 WHERE id = 1",
+            )
```

Comment:
This should work on all backends so I moved `test_last_executed_query_with_duplicate_params` to the `backends.tests`.